resources/ext/cli_defaults/cli-defaults.sh.erb

```diff
@@ -7,4 +7,4 @@ if [ ! -e "$JRUBY_JAR" ]; then
   return 1
 fi
 
-CLASSPATH="${CLASSPATH}:${JRUBY_JAR}"
+CLASSPATH="${CLASSPATH}:${JRUBY_JAR}:/opt/puppetlabs/server/data/puppetserver/jars/*"
```

Comment:
Probably couldn't hurt. Does this sometimes get used in the context of something in PE which causes the name to change, maybe?